src/main/webapp/components/NoteGridCard.vue

```diff
@@ -36,6 +36,12 @@
         // TODO: Generate a thumbnail of the pdf
         let source = 'assets/notes.png';
         return {backgroundImage:`url(${source})`};
+      },
+      dateString: function() {
+        if(this.dateCreated) {
+          return this.dateCreated.toDateString();
+        }
+        return "";
```

Comment:
Is `dateString` necessary because I think `dateCreated` displays the formatted date already?

Reply:
Also, I think I forgot to add it but same thing in NotePreview.vue.

Reply:
I'm going to make a change to dateCreated to ensure it's a parsed Date() object, since
- A date object is what the servlet responds with
- It's better for sorting notes by date.
The formatting should be done at the last possible moment.